deploy/openshift/templates/README.md

```diff
@@ -0,0 +1,155 @@
+## Context
+
+All commands are executed in the directory where this README is located.
+
+## Templates and Parameters
+
+Templates are provided with a set of predefined params which you can override with with `-p key=value`. If you miss envs and parameters, you can add them to your template.
+
+IMPORTANT! **ROUTING_SUFFIX** value in the below commands `$(minishift ip).nip.io` works for MiniShift only. You MUST provide **own routing suffix** when deploying to OCP cluster.
+
+## Why many templates?
+
+Since there are several Che flavors and configuration options, it is not possible to list all objects in one template, while duplicating templates makes it more difficult to maintain them.
+
+## Che server logs persistence
+
+All of the below commands create a PVC for Che server and Che deployment uses this PVC.
+
+It is possible to avoid using PVC for Che server - just skip creating pvc `oc apply -f pvc/che-server-pvc.yaml` and `oc set volume dc/che --add -m /data --name=che-data-volume --claim-name=che-data-volume` commands.
+Without PVC Che deployment can use Rolling update strategy: `-p STRATEGY=Rolling` that causes zero downtime when updating Che server.
+However, in this case, if you need to persist logs you need to use solutions like Kibana.
+
+## Creating Workspace Objects
+
+By default, che SA is used to create workspace objects in the same namespace with Che server. This is configurable - you may provide login and password or token, and set
```

Comment:
FYI sometimes Che is written `che` sometimes `Che`

Reply:
is there any word expected after `and set` ?

Reply:
Fixed